platform/qemu.go

```diff
@@ -210,7 +211,12 @@ func setupDisk(additionalOptions ...string) (*os.File, error) {
 	if err := qemuImg.Run(); err != nil {
 		return nil, err
 	}
-
+	if len(confPath) > 0 {
+		err = setupIgnition(confPath, dstFileName)
```

Comment:
Nit: this is usually written as a single statement, like `if err := setupIgnition(...); err != nil { ... }`.